Modules/_ctypes/cfield.c

```diff
@@ -56,51 +55,42 @@ Py_ssize_t LOW_BIT(Py_ssize_t offset);
 @classmethod
 _ctypes.CField.__new__ as PyCField_new
 
+    *
     name: object(subclass_of='&PyUnicode_Type')
     type as proto: object
-    size: Py_ssize_t
-    offset: Py_ssize_t
+    byte_size: Py_ssize_t
+    byte_offset: Py_ssize_t
     index: Py_ssize_t
+    for_big_endian: bool
+    _internal_use: bool
     bit_size as bit_size_obj: object = None
+    bit_offset as bit_offset_obj: object = None
 
 [clinic start generated code]*/
 
 static PyObject *
 PyCField_new_impl(PyTypeObject *type, PyObject *name, PyObject *proto,
-                  Py_ssize_t size, Py_ssize_t offset, Py_ssize_t index,
-                  PyObject *bit_size_obj)
-/*[clinic end generated code: output=43649ef9157c5f58 input=3d813f56373c4caa]*/
+                  Py_ssize_t byte_size, Py_ssize_t byte_offset,
+                  Py_ssize_t index, int for_big_endian, int _internal_use,
+                  PyObject *bit_size_obj, PyObject *bit_offset_obj)
+/*[clinic end generated code: output=79505dee1dad9b8e input=a6376bdec96976b8]*/
 {
     CFieldObject* self = NULL;
-    if (size < 0) {
-        PyErr_Format(PyExc_ValueError,
-                     "size of field %R must not be negative, got %zd",
-                     name, size);
+
+    if (!_internal_use) {
+        // Do not instantiate outside ctypes, yet.
+        // The constructor is internal API and may change without warning.
+        PyErr_Format(PyExc_TypeError, "Cannot create %T object.", type);
```

Comment:
```suggestion
        PyErr_Format(PyExc_TypeError, "cannot create %T object", type);
```

Considering all exceptions messages by a lowercase letter and don't end with a period, let's be consistent here as well